### ISSUE TITLE
Water and Power inverted on PC sheet

### ISSUE THREAD

**Issue by JhaelenTerear** [pwatson100/alienrpg#73]
Hello,

All is in the title : on the PC sheet, labels are inverted for Water and Power.

**Comment by pwatson100** [comment]
I have just done a fresh installation and am not seeing this issue.  Do you have any of the Alien UI modules installed?   If so could you disable them and try again.

**Comment by pwatson100** [comment]
I have made him aware and he is going to update the module.

**Comment by JhaelenTerear** [comment]
Yeah, sorry, I checked this this morning when I get out of bed, and you're right, it's Alien RPG CRT UI the culprit :).
